src/cultionet/models/enums.py

```diff
@@ -0,0 +1,11 @@
+import enum
+
+
+class ModelTypes(enum.Enum):
```

Comment:
This enum class might be useful here, or elsewhere in the codebase: https://docs.python.org/3/library/enum.html#enum.StrEnum

Works nicely when you want to map enum values to strings of their names.